src/globus_sdk/_testing/models.py

```diff
@@ -40,14 +59,44 @@ class RegisteredResponse:
     def __init__(
         self,
         *,
+        # path and service are glbous-sdk specific
+        # in `responses`, these are just `url`
         path: str,
-        service: str | None = None,
-        method: str = responses.GET,
+        service: (
+            Literal[
+                "auth",
+                "nexus",
+                "transfer",
+                "search",
+                "gcs",
+                "groups",
+                "timer",
+                "flows",
+                "compute",
+            ]
+            | None
+        ) = None,
+        # method derives from `responses.BaseResponse`
+        method: Literal["GET", "PUT", "POST", "PATCH", "HEAD", "DELETE"] = "GET",
+        # these parameters derive from `response.Response`
+        body: str | None = None,
+        content_type: str | None = None,
         headers: dict[str, str] | None = None,
-        metadata: dict[str, t.Any] | None = None,
         json: None | list[t.Any] | dict[str, t.Any] = None,
-        body: str | None = None,
-        **kwargs: t.Any,
+        status: int = 200,
+        stream: bool | None = None,
+        # match derives from `responses.BaseResponse`; it has a default of `()` there
+        match: t.Sequence[t.Callable[..., tuple[bool, str]]] = (),
```

Comment:
Because the implementation's default is noted here, should this be updated if it ever changes in `responses`? If not, then we're choosing a default for our own purposes, and the implementation's default doesn't matter.

Reply:
I think I'm going to change this, since the default in `responses` could change and we wouldn't track it with this behavior.
Instead, let's use `t.Sequence[...] | None = None` and then at the usage site (below) we'll only pass it if it has a non-`None` value.

Reply:
I also did my best to clarify the comment wording. `BaseResponse` vs `Response` only really matters if you are looking at the _current_ `responses` source code and trying to understand where the arguments for `Response` are defined.

Since that kind of examination is a rare activity, and `responses` could change, it now all says that we pass values to `responses.Response`, which is more robust (as a statement) to changes in `responses`.